express-delete/index.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res, next) => {
+  const newarray = [];
+  for (const prop in grades) {
+    const result = grades[prop];
+    newarray.push(result);
+  }
+  res.json(newarray);
+  next();
```

Comment:
We only want to call `next` if there is a next thing to do. Once we send a response with `res.json` the request has completed and there is nothing else to do or that should be done. So we don't want to use `next` here.

Reply:
so in the case of calling next, that would be if we had multiple of the same app.methods?